packages/next/package.json

```diff
@@ -209,6 +209,7 @@
     "ci-info": "watson/ci-info#f43f6a1cefff47fb361c88cf4b943fdbcaafe540",
     "cli-select": "1.1.2",
     "client-only": "0.0.1",
+    "commander": "12.0.0",
```

Comment:
Could we bump CNA's commander to to match here or are there breaking changes that warrant separate PR? 

x-ref: https://github.com/vercel/next.js/blob/8a2460ccb1d5dc480eb5d0f501daf8f839adb286/packages/create-next-app/package.json#L42

Reply:
Yeah I need to fix CNA as well. Separate <picture data-single-emoji=":pr:" title=":pr:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/pr/876af7e7d65e1595.png" alt=":pr:" align="absmiddle"></picture>